chemprop/v2/cli/train.py

```diff
@@ -466,25 +468,9 @@ def process_train_args(args: Namespace) -> Namespace:
     with open(args.data_path) as f:
         args.header = next(csv.reader(f))
```

Comment:
I think this can be replaced with pandas too

Reply:
will actually delete because it's not used anywhere.